### ISSUE TITLE
How to alternate between training and validating? 

### ISSUE THREAD

**Issue by YiruS** [junyanz/pytorch-CycleGAN-and-pix2pix#766]
Hi,

I'd like to train some epochs (say every 10 epochs) and evaluate the model on validation dataset. However, the model is the subclass of BaseModel(), which seems no support for switching between training mode and val mode. For example, batch norm, or dropout has different mode. Would there be any suggestions?

Thanks!

**Comment by YiruS** [comment]
Problem solved, close it.

**Comment by hyecheol123** [comment]
@YiruS May I ask you how did you solve this problem?

**Comment by chenglingjing** [comment]
@YiruS @hyecheol123 你好，请问这个问题怎么解决呢
